util/id_generator.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+// IDGenerator util class used for generate auto-increasing id
+type IDGenerator struct {
```

Comment:
seems we can simulate what the planID did to avoid introducing another file that has the same functionality.
```
// PlanID is the unique id of logical and physical plan 
PlanID atomic.Int32
```

Reply:
We don't have a id util to get auto increasing id. Also the plan id is the unique id of plan but I need a runtimefilter id which is unique id of runtime filter. (There are many runtime filters in one plan)

Reply:
I mean 
```
RuntimeFilterID atomic.Int32
usage like: ctx.GetSessionVars().RuntimeFilterID.Add(1)
```
it's ok, not a strong requirement

Reply:
Oh, This is indeed a method